docker-compose.prod.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+
+services:
+  firefly-app:
+    container_name: firefly-iii
+    environment:
+      - FF_APP_ENV=production
+    volumes:
+      - ./storage:/var/www/firefly-iii/storage
```

Comment:
That's strange, I changed this to `firefly-app-storage:/var/www/firefly-iii/storage` and added the volume at the end of the file (like the database one) and this is the content of storage the first time run the container:
```
root@c2d7dae5bc11:/var/www/firefly-iii# ls storage/   
app  build  database  debugbar	export	framework  logs  upload

root@c2d7dae5bc11:/var/www/firefly-iii# ls storage/framework/
cache  sessions  views
```

Don't get me wrong, the current line works fine too. I always try make the compose files, in a way I would be able to just copy-paste them in a server folder and just run them from there without cloning the whole app repository.
(However, to do this with firefly-iii we also need to push the image to the docker-hub)